src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -36,7 +36,7 @@
     <!-- DependenciesTree capability lights up a Dependencies tree node and it's sub node providers-->
     <ProjectCapability Include="DependenciesTree" />
     <ProjectCapability Include="LaunchProfiles" />
-
+    <ProjectCapability Include="NoGeneralDependentFileIcon"/>
```

Comment:
I did create a merge request https://github.com/Microsoft/VSProjectSystem/pull/161 , could you please merge it? I don't have write access.